schedule/qam/common/mau-extratests2.yaml

```diff
@@ -125,4 +136,10 @@ conditional_schedule:
         - console/year_2038_detection
       aarch64:
         - console/year_2038_detection
+  arch_specific15_sp6:
```

Comment:
I would just rename `arch_specific15_sp5` to `arch_specific15_sp5+` and use that,
no need to create copy of it